src/components/Form/Form.tsx

```diff
@@ -1 +1,36 @@
-export {}
+import React from 'react';
+import './Form.css';
+
+
+interface userSentimentProps {
+  userSentiment: (newSentiment: number) => void
+}
+
+const Form: React.FC<userSentimentProps> = props => {
+  const assignSentiment = (event: React.MouseEvent<HTMLButtonElement>) => {
+      event.preventDefault();
+      const button: HTMLButtonElement = event.currentTarget;
+      props.userSentiment(Number(button.name));
+  }
+
+  return(
+    <div className="form-container">
+      <section className="questionnaire">
+        <h2>how are you feeling today?</h2>
+        <article className="sentiment-selection">
+          <button className="happy-btn" name="1" onClick={assignSentiment}></button>
+          <button className="neutral-btn" name="0" onClick={assignSentiment}></button>
+          <button className="sad-btn" name="-1" onClick={assignSentiment}></button>
+        </article>
+        <article className="wellnews-synopsis">
```

Comment:
Once we turn this into a button, should this be its own component that appears across all routes?

Reply:
Good idea! Perhaps with a different titles, like 'Why we ask?' on the quiz and 'How we sort your feed' on the news feed.